package.json

```diff
@@ -12,7 +12,6 @@
   "license": "MIT",
   "repository": "https://github.com/Neufund/Typechain",
   "devDependencies": {
-    "@types/bignumber.js": "^4.0.3",
```

Comment:
@krzkaczor do we need typings for BigNumber when they are provided internally by `bignumber.js` through `types` package.json field?

Reply:
If it still compiles then probably not :D I don't remember the original reason to add it.